packages/shared/routes/dashboard/wallet/views/Receive.svelte

```diff
@@ -13,14 +13,15 @@
     const generateNewAddress = (): void => {
         onGenerateAddress($selectedAccount.id)
     }
+
     const handleCloseClick = (): void => {
         $accountRouter.previous()
     }
 </script>
 
-<div class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0 p-6">
+<div class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0 p-8">
     <div class="w-full flex flex-row justify-between items-center">
-        <div class="w-full flex flex-row space-x-4 items-center">
+        <div class="w-full flex flex-row space-x-4 items-center justify-center">
```

Comment:
I think we should leave this to the left, as to match all of the other dashboard tiles.

Reply:
Oops, good catch! I thought it looked off and that I had changed it to go to the left. Apparently not! :sweat_smile: . Will change it back!